src/main/java/frc/shooter/ShooterSensors.java

```diff
@@ -0,0 +1,43 @@
+package frc.shooter;
+
+import frc.sensors.Limelight;
+import frc.shooter.MathHelpers;
+import frc.drives.DrivesSensors;
+
+public class ShooterSensors implements ShooterSensorsInterfeace 
+{
+	Limelight lightSensor; //Used for accessing member methods of the limelight.
+	DrivesSensors driveSensor; //Used for accessing member methods of the drive sensors.
+	double cameraToRobotAngle;
+	public ShooterSensors(double cameraToRobotAngle, Limelight light, DrivesSensors drive)
+	{
+		this.lightSensor = light;
+		this.driveSensor = drive;
+		this.cameraToRobotAngle = cameraToRobotAngle;
+	}
+	public double getDistanceToTarget() 
+	{
+		return this.lightSensor.getDistanceFromTarget();
+	}
+	
+	public double getAngleToTarget() 
+	{
+		return this.lightSensor.getAngleFromTarget();
+	}
+	
+	public double getShooterAngleToRobot()

```

Comment:
This method is intended to return the current position of the turret using the encoder.
Need to create an instance of an encoder and call the .getDistance method 
I see how the naming could be confusing.